lib/control.rb

```diff
@@ -0,0 +1,112 @@
+class Control
+  require_relative './plateu'
+  require_relative './rover'
+  require_relative './houston_command'
+  attr_accessor :x, :y, :plateu, :rover, :commands, :command_array
+
+  def command_rover(string)
+    @command_array = string
+    set_up_plateu_and_delete
+    set_up_rover_and_send_commands
+     while check_for_commands(@command_array)
+       reset_rover_and_send_commands
+     end
+  end
+
+  def set_up_plateu_and_delete
+    set_up_plateu(@command_array)
+    delete_first_two(@command_array)
+  end
+
+  def set_up_rover_and_send_commands
```

Comment:
Feels like this method has multiple responsibilities.  Partly, this is because it has an and in the name, which is usually to be avoided.  It also does further things like showing the rover.